src-java/blue-green/src/main/java/org/openkilda/bluegreen/kafka/interceptors/VersioningConsumerInterceptor.java

```diff
@@ -121,6 +122,7 @@ public void configure(Map<String, ?> configs) {
         connectionString = getValue(configs, CONSUMER_ZOOKEEPER_CONNECTION_STRING_PROPERTY, String.class);
         runId = getValue(configs, CONSUMER_RUN_ID_PROPERTY, String.class);
         componentName = getValue(configs, CONSUMER_COMPONENT_NAME_PROPERTY, String.class);
+        reconnectDelayMs = getValue(configs, CONSUMER_ZOOKEEPER_RECONNECTION_DELAY_PROPERTY, Long.class);
```

Comment:
I used `getValue` here because 3 props above are mandatory. ZkClient can't work without them. 
But `reconnectDelayMs` is optional, if it will be 0 we will use default value. So you can use just 
`configs.getOrDefault(CONSUMER_ZOOKEEPER_RECONNECTION_DELAY_PROPERTY, 0)`